### PR TITLE
docs: Added all-contributions instructions closes #204

### LINKED ISSUE #204: Add all-contributors bot to repo
### User Story: 

As a software developer, I want this repository to utilise the all-contributors bot so that all types of contributions made by contributors to the project are fairly recognised.

### Acceptance Criteria:

1) All-Contributors bot is installed on this repository
2) Contributions to the project are visible in README

### Notes
https://allcontributors.org/

### PR DESCRIPTION
**Pull request by AlexanderTheGrape**
- [ ] The pull request is complete according to the following criteria:
  - [ ] Acceptance criteria have been met
  - [x] The documentation is kept up-to-date
  - [ ] Comprehensive tests (if applicable) have been generated and all pass.
  - [ ] The pull request describes the changes that have been made, and enough information is present in the description for any developer to understand what has changed
  - [ ] Commits have been squashed (or will be on merge).
  - [ ] The branch name is descriptive and follows the pull request title format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [ ] The pull request title is of the following format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [ ] The description uses [github syntax](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) to link to the issue. E,g Resolves se701g2/Doto#{Number}
  - [x] At least two reviewers assigned. One of which must be the assigner of the issue.
  - [ ] If there are merge conflicts, run git rebase as opposed to git merge with master.

### CODE REVIEW

**Comment by AlexanderTheGrape** [comment]
Note that this pull request is tied to this pull request https://github.com/se701g2/Doto/pull/266
and this issue https://github.com/se701g2/Doto/issues/204

File: CONTRIBUTIONS.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+If you'd like your contributions to be displayed in the repo readme, do the following:
+
+In an issue or pull request, simply comment the following: 
+    @all-contributors please add @<github username> for <contribution>
```
Comment (KimberleyEvans-Parker) [inline CONTRIBUTIONS.md:4]: If this is likely to be viewed as an md, then it looks like anything inside <> gets treated as a comment and removed

Comment (Matteas-Eden) [reply CONTRIBUTIONS.md:4]: Adding on this, I suggest using `code blocks`. For example;
`@all-contributors please add @<github username> for <contribution>`

Comment (AlexanderTheGrape) [reply CONTRIBUTIONS.md]: Thanks for spotting those, looks like I forgot to check it! I've made those changes now.

**Pull request merged**